Source/BugsnagNotifier.m

```diff
@@ -399,14 +399,19 @@ - (void)start {
 
     _started = YES;
     // autoDetectErrors disables all unhandled event reporting
-    BOOL configuredToReportOOMs = self.configuration.reportOOMs && self.configuration.autoDetectErrors;
+    BOOL configuredToReportOOMs = self.configuration.reportOOMs
```

Comment:
The new error types should replace `reportOOMs`, right?

Reply:
I wasn't sure.  Do we want to deprecate this in the first instance rather than excise it completely?

Reply:
That would be fine too, though the setter/getter would need to proxy to enabled error types rather than leaving both in place. In the current configuration, there isn't a way to determine precedence since the property can't be unset and is off by default in some cases.

Reply:
Excised as an exercise in simplicity.